main.go

```diff
@@ -124,6 +142,18 @@ type PrometheusTaskInfo struct {
 	Labels  labels   `yaml:"labels"`
 }
 
+// Regex used for normalizing label names
+var labelRE = regexp.MustCompile("[^a-zA-Z0-9_]")
```

Comment:
why define this outside globally and not within the normalizeLabelName func?

Reply:
I think it wouldn't make a significant difference. I can also define this inside `normalizeLabelName()`.

Reply:
OK fine for now, but keeping variable inside functions they are used, helps in readability, and also limits the number of globals